tests/tracked_struct_not_validated_due_to_durability.rs

```diff
@@ -0,0 +1,108 @@
+use salsa::{Durability, Setter};
```

Comment:
Other tests have a module docstring that explains the test case. The PR summary (or parts of it) would make a good intro to this test